src/redux/actions/data.js

```diff
@@ -17,20 +17,12 @@ export const setWidth = width => ({
     payload: width,
 })
 
-// https://virtual-sports-yi3j9.ondigitalocean.app/
-
-export const fetchData = () => dispatch => {
+export const fetchData = token => dispatch => {
     dispatch(setLoaded(false))
-
-    // TODO: just for test - remove on production
-    setTimeout(() => {
-        dispatch(setData(response))
-        dispatch(setLoaded(true))
-    }, 2000)
-
-    // TODO
-    /* getDataFromSeverFunction().then(({ data }) => {
-        // TODO: create an object with 4 arrays from the response
-        dispatch(setData(data))
-    }) */
+    getDataFromSeverFunction(token)
+        .then(data => data.json())
+        .then(body => {
+            dispatch(setData(body))
+            dispatch(setLoaded(true))
```

Comment:
I fixed that in my previous PRs, there's no need to manually set `isLoaded: true` here. It should be done via actions

Reply:
https://github.com/Virtual-Sports/Web/blob/8b2046b599f78d81146bb6235aa8374b6b129094/src/redux/reducers/data.js#L25